cli/src/main/kotlin/commands/ScannerCommand.kt

```diff
@@ -120,10 +120,10 @@ class ScannerCommand : CliktCommand(name = "scan", help = "Run external license
 
     private val globalOptionsForSubcommands by requireObject<GlobalOptions>()
 
-    private fun configureScanner(config: ScannerConfiguration): Scanner {
-        ScanResultsStorage.configure(config)
+    private fun configureScanner(scannerConfig: ScannerConfiguration): Scanner {
```

Comment:
Commit message title: The scope should be "scanner" instead of just "LocalScanner".